test/XRP/helpers/xrp-test-utils.ts

```diff
@@ -1,12 +1,20 @@
 import { GetAccountTransactionHistoryResponse } from '../../../src/XRP/Generated/web/org/xrpl/rpc/v1/get_account_transaction_history_pb'
 import XrpTransaction from '../../../src/XRP/model/xrp-transaction'
-import { XrplNetwork } from 'xpring-common-js'
+import { Wallet, XrplNetwork } from 'xpring-common-js'
+import { XrpUtils, XrpError, XrpErrorType } from '../../../src/XRP'
 import XrpMemo from '../../../src/XRP/model/xrp-memo'
+import bigInt from 'big-integer'
+import XrpClient from '../../../src/XRP/xrp-client'
+import axios from 'axios'
 
 /**
  * Convenience class for utility functions used in test cases for XrpClient infrastructure.
  */
 export default class XRPTestUtils {
+  static sleep = async (milliseconds: number): Promise<void> => {
+    return new Promise((resolve) => setTimeout(resolve, milliseconds))
+  }
```

Comment:
nit: you don't need an expression+return here; ie

`...Promise<void> => new Promise(....`